src/main/java/application/bookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,22 @@
 package application.bookstore.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    private static final long MIN_VALUE = 0;
+
+    @NotNull
     private String title;
+    @NotNull
     private String author;
+    @NotNull
     private String isbn;
+    @NotNull
+    @Min(MIN_VALUE)
     private BigDecimal price;
     private String description;
```

Comment:
I guess you can make validation more detail, you can validate description by size or something else for example